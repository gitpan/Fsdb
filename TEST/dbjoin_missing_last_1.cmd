prog='dbjoin'
args='-S -e 0 -a --input TEST/dbjoin_missing_last_1.in-2 low'
cmp='diff -cb '
