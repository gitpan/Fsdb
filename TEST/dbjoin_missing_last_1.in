#h low histogram
1       47
2       82
3       97
4       85
5       59
6       36
7       10
8       7
