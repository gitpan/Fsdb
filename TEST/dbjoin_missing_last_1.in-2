#h low noshare
1       7
2       39
3       86
4       137
5       150
6       101
7       38
8       13
9       6
