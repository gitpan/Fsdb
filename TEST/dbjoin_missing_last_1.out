#fsdb low histogram noshare
1	47	7
2	82	39
3	97	86
4	85	137
5	59	150
6	36	101
7	10	38
8	7	13
9	0	6
#   | dbjoin -S -e 0 -a --input TEST/dbjoin_missing_last_1.in-2 low
